### PR TITLE
Blog pagination fixes

### PR DESCRIPTION
**Pull request by simonbaird**
(See details in commit messages.)

Ref: [EC-491](https://issues.redhat.com/browse/EC-491)

### CODE REVIEW

File: website/config/_default/hugo.toml [hunk 1]
```diff
@@ -6,4 +6,4 @@ publishDir = '../public'
 copyright = "Red Hat, Inc. All rights reserved."
 relativeURLs = true
 staticDir = ["static", "antora"]
-paginate = 2
+paginate = 12
```
Comment (lcarva) [inline website/config/_default/hugo.toml:9]: Nice

File: website/layouts/index.html [hunk 2]
```diff
@@ -55,15 +55,21 @@
         <div class="blog-excerpts">
           <h2>Recently Published</h2>
           <article class="doc">
-            {{ $paginator := .Paginate (where .Site.RegularPages "Type" "in" .Site.Params.mainSections) }}
+            {{ $paginator := .Paginate (where .Site.RegularPages "Type" "in" .Site.Params.mainSections) 2 }}
```
Comment (lcarva) [inline website/layouts/index.html]: What's the `2` for?

**Review by lcarva — APPROVED** [review]

File: website/layouts/index.html (hunk 2 above)
Comment (simonbaird) [reply website/layouts/index.html]: I'm fuzzy on how it works, but IIUC this part only for the home page article excerpts. There we show the top 2.

File: website/layouts/index.html [hunk 3]
```diff
@@ -55,15 +55,21 @@
...
 
+            {{/* This div is not really for pagination but the CSS does what we want and also I'm lazy */}}
+            <div class="pagination-centered">
+              <div class="pagination pagination-default" style="padding: 3em 0; opacity: 50%;">
+                {{ range .Site.Menus.main }}{{ if eq .Name "Blog" }}
+                  <a href="{{.URL}}">More articles &hellip;</a>
+                {{ end }}{{ end }}
+              </div>
+            </div>
+
```
Comment (zregvart) [inline website/layouts/index.html]: Could be simpler, something like might work:

```suggestion
            {{ $posts := where .Site.RegularPages "Type" "in" .Site.Params.mainSections }}
            {{ range $posts.Limit 2 }}
              {{ partial "post-preview.html" . }}
            {{ end }}

            {{/* This div is not really for pagination but the CSS does what we want and also I'm lazy */}}
            <div class="pagination-centered">
              <div class="pagination pagination-default" style="padding: 3em 0; opacity: 50%;">
                <a href="/posts">More articles &hellip;</a>
              </div>
            </div>

```

Comment (simonbaird) [reply website/layouts/index.html]: Re `<a href="/posts">More articles &hellip;</a>` I was trying to avoid hard coding the url..

Comment (simonbaird) [reply website/layouts/index.html]: Included in latest revision.

Comment (simonbaird) [reply website/layouts/index.html]: Thanks!

**Review by zregvart — APPROVED** [review]

**Comment by simonbaird** [comment]
Will stop tinkering with the whitespace and merge it now...

**Pull request merged**